sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/RemoteCryptographyClient.cs

```diff
@@ -63,21 +31,7 @@ public RemoteCryptographyClient(Uri keyId, TokenCredential credential, Cryptogra
 
         internal KeyVaultPipeline Pipeline { get; }
 
-        /// <summary>
-        /// 
-        /// </summary>
-        public Uri KeyId { get; }
-
-        /// <summary>
-        /// 
-        /// </summary>
-        /// <param name="algorithm"></param>
-        /// <param name="plaintext"></param>
-        /// <param name="iv"></param>
-        /// <param name="authenticationData"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public virtual async Task<Response<EncryptResult>> EncryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext, byte[] iv = default, byte[] authenticationData = default, CancellationToken cancellationToken = default)
+        private async Task<Response<EncryptResult>> EncryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext, byte[] iv = default, byte[] authenticationData = default, CancellationToken cancellationToken = default)
```

Comment:
>private [](start = 8, length = 7)

I'd prefer not to change all these to private and make callers call through the interface.  Especially since we might make this public at some point.